client/js/lounge.js

```diff
@@ -474,6 +480,9 @@ $(function() {
 	});
 
 	const contextMenuActions = {
+		join: function(itemData) {
+			$(`#join-channel-${itemData}`).closest(".network").find("button.add-channel").click();
```

Comment:
Could we change this to ... not do this but call the function on the click handler instead? We keep ending up with issues because we rely on UI elements and then when we change or move those elements it breaks other things.